pkg/webhook/validation/dynakube/dynakube_name_test.go

```diff
@@ -24,3 +25,29 @@ func TestNameStartsWithDigit(t *testing.T) {
 		})
 	})
 }
+
+func TestNameTooLong(t *testing.T) {
+	t.Run(`normal name`, func(t *testing.T) {
```

Comment:
I prefer the table drive test, because we want to test
- 39, 40, 41 
😄

Reply:
the problem with that is the little helper functions `assertDeniedResponse` and `assertAllowedResponse` are not that friendly to table tests

I will give it a try, but it might get ugly 😬

Reply:
oh i see okey

Reply:
you can keep it as is. But in general would be nice to test MAX_VALUE -1 , MAX_VALUE, MAX_VALUE + 1

Reply:
I was wrong, I could make it work 😉